docs/source/release_notes/release-notes.rst

```diff
@@ -130,6 +130,10 @@ develop
          - Removed unnecessary memory allocations in the lock refresher, and in several other classes, by using Lists.partition(...) instead of Iterables.partition(...).
            (`Pull Request <https://github.com/palantir/atlasdb/pull/3918>`__)
 
+    *    - |improved|
+         - AtlasDB now logs diagnostic information about uses of the request batching timestamp service.
```

Comment:
I guess this should have been write batching transaction service instead of request batching timestamp service, right?

Reply:
Yeah, made an error there. I've expanded the scope of this to hit all the batchers, so this has been reworded.